src/pages/Facility/settings/organizations/FacilityOrganizationIndex.tsx

```diff
@@ -43,16 +46,152 @@ export default function FacilityOrganizationIndex({
   facilityId: string;
 }) {
   const [expandedRows, setExpandedRows] = useState<Record<string, boolean>>({});
+  const [searchQuery, setSearchQuery] = useState("");
   const { t } = useTranslation();
+  const isMobile = useBreakpoints({ default: true, sm: false });
+
   const { data, isLoading } = useQuery({
     queryKey: ["facilityOrganization", "list", facilityId],
     queryFn: query.paginated(routes.facilityOrganization.list, {
       pathParams: { facilityId },
     }),
     enabled: !!facilityId,
   });
+
   const tableData = data?.results || [];
-  const isMobile = useBreakpoints({ default: true, sm: false });
+
+  const [{ childrenMap, topLevelOrganizations }, setOrganizationHierarchy] =
+    useState<{
+      childrenMap: Map<string, (typeof tableData)[0][]>;
```

Comment:
```suggestion
      childrenMap: Map<string, FacilityOrganization[]>;
```
Replace other instances as well.